### PR TITLE
Pass through additional psycopg2 kwargs

### PR DESCRIPTION
**Pull request by jklukas**
Necessary for fine-grained SSL config.

### CODE REVIEW

**Comment by jklukas** [comment]
Hi, kf.

**Comment by jklukas** [comment]
I'm figuring out the test errors. It's travis-related rather than code-related.

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,12 @@ language: python
...
   matrix:
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=py35
-  - TOX_ENV=docs
-  - TOX_ENV=lint
+    - TOX_ENV=py27
+    - TOX_ENV=py34
+    - TOX_ENV=py35
```
Comment (andharris) [inline .travis.yml]: This formatting diff made me think, we should at 3.6 to this. Thoughts

Comment (jklukas) [reply .travis.yml]: Travis is failing on py26, py33, and py36. It's not entirely clear to me why, but the quick solution that gets tests passing for the moment doesn't allow us to test on py36. I'd be happy to see a separate PR that investigates how to get py36 working with travis.

**Pull request merged**